specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/batchInference.json

```diff
@@ -0,0 +1,1712 @@
+        }
+      },
+      "discriminator": "computeType"
+    },
+    "BatchDeploymentSettings": {
+      "description": "Batch inference specific settings per deployment.",
+      "type": "object",
+      "properties": {
+        "computeId": {
+          "description": "Compute artifact Identifier",
+          "type": "string"
+        },
+        "errorThreshold": {
+          "format": "int32",
+          "description": "Error threshold, if the error count for the entire input goes above this value,\r\nthe batch inference will be aborted. Range is [-1, int.MaxValue]\r\n-1 value indicates, ignore all failures during batch inference\r\nFor FileDataset count of file failures\r\nFor TabularDataset, this is the count of record failures",
+          "type": "integer"
+        },
+        "retrySettings": {
+          "$ref": "#/definitions/BatchRetrySettings"
+        },
+        "partitioningScheme": {
+          "$ref": "#/definitions/BatchPartitioningScheme"
+        },
+        "loggingLevel": {
+          "$ref": "#/definitions/BatchLoggingLevel"
+        },
+        "outputConfiguration": {
+          "$ref": "#/definitions/BatchOutputConfiguration"
+        }
+      }
+    },
+    "InteractionEndpoints": {
+      "description": "Interaction endpoints",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/InteractionEndpoint"
+      }
+    },
+    "InteractionEndpoint": {
+      "description": "Interaction endpoint",
+      "properties": {
+        "jobEndpoint": {
+          "$ref": "#/definitions/JobEndpoint"
+        },
+        "link": {
+          "type": "string"
+        }
+      }
+    },
+    "JobOutput": {
+      "description": "Output of a job.",
+      "type": "object",
+      "properties": {
+        "datastoreId": {
+          "description": "Datastore Identifier",
+          "type": "string"
+        },
+        "path": {
+          "description": "Path inside the datastore",
+          "type": "string"
+        }
+      }
+    },
+    "JobEndpoint": {
+      "description": "Job endpoint.",
+      "enum": [
+        "Tracking",
+        "Studio",
+        "Grafana",
+        "Tensorboard"
+      ],
+      "type": "string"
+    },
+    "JobStatus": {
+      "description": "The status of a job.",
+      "enum": [
+        "NotStarted",
+        "Starting",
+        "Provisioning",
+        "Preparing",
+        "Queued",
+        "Running",
+        "Finalizing",
+        "CancelRequested",
+        "Completed",
+        "Failed",
+        "Canceled",
+        "NotResponding"
+      ],
+      "type": "string"
+    },
+    "BatchJob": {
+      "description": "Batch job inference settings.",
+      "type": "object",
+      "properties": {
+        "scaleSettings": {
+          "$ref": "#/definitions/DeploymentScaleSettings"
+        },
+        "datasetId": {
+          "description": "Identifier for input dataset.",
+          "type": "string"
+        },
+        "tags": {
```

Comment:
"tags" should be on "BatchJobResource" (the top level of the resource's JSON body) if the resource supports tags, not in this definition.

Reply:
BatchJobs are proxy resources and for the proxy resources, the envelope (top-level) does not have tags, correct?

Reply:
"tags" are only supported for tracked resources which is why proxy resources don't have them.  It should not be present if these are proxy resources

Reply:
Removing them.